garrysmod/gamemodes/base/gamemode/cl_voice.lua

```diff
@@ -97,6 +97,8 @@ function GM:PlayerStartVoice( ply )
 
 	if ( !IsValid( ply ) ) then return end
 
+	if hook.Run("ShouldDrawPlayerVoicePanel", ply) == false then return end

```

Comment:
Use hook.Call and pass ``self`` as the gamemode table. Replace GAMEMODE with self above.

Reply:
Also needs to match the codebase style. Spaces around between parentheses and body, and needs parentheses around the if-condition.

Reply:
Done